src/ui/reference/_colors.scss

```diff
@@ -1,11 +1,34 @@
 /* stylelint-disable color-no-hex */
-$black: black;
-$dark-grey: #484848;
-$grey: #757575;
-$light-grey: #EBEBEB;
 
-$red: #A43030;
+:root {
+  // Button
+  --button-bg: black;
 
-$default-box-shadow: 0 0 5px $light-grey;
+  // Dropdown
+  --dropdown-text-color: black;
+  --dropdown-header-button-bg: #ebebeb;
+  --dropdown-selected-item-bg: #ebebeb;
```

Comment:
I think this file will increasingly get larger as we add more components to the UI-kit. Can we define the variables in respective SCSS files for the components? For example, dropdown variables can be defined inside `_dropdown.scss`.

Reply:
Filed an issue https://github.com/Hipo/react-ui-toolkit/issues/11